specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2024-05-01-preview/examples/JobExecution_Get.json

```diff
@@ -0,0 +1,50 @@
+{
+  "parameters": {
+    "api-version": "2024-05-01-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "myResourceGroup",
+    "serviceName": "myservice",
+    "jobName": "test-job",
+    "jobExecutionName": "test-job-execution"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "name": "test-job-execution",
+        "properties": {
+          "resourceRequests": {
+            "cpu": "1",
+            "memory": "2Gi"
+          },
+          "template": {
+            "args": [
+              "arg1"
+            ],
+            "environmentVariables": [
+              {
+                "name": "key1",
+                "value": "value1"
+              },
+              {
+                "name": "env2",
+                "value": "value2"
+              },
+              {
+                "name": "secretKey1"
+              }
+            ]
+          }
+        },
+        "jobSnapshot": {
```

Comment:
I think that backend can decide which properties to return. In job case, `provisioningState` won't be returned in `jobSnapshot`, but `source` will since user needs to know which source they deployed for the execution.

Reply:
BTW, the SDK will still have these properties.